pkg/engine/executor/dataobjscan.go

```diff
@@ -102,6 +102,24 @@ func (s *dataobjScan) init(ctx context.Context) error {
 			return fmt.Errorf("opening logs section: %w", err)
 		}
 
+		{
+			si := sec.SortInfo().GetColumnSorts()
+			if len(si) == 0 {
+				return fmt.Errorf("missing sort order info")
```

Comment:
This means that we cannot use data objects that we have built so far, right?
Should we instead assume [Timestamp DESC, StreamID ASC]?